misc/run_length_encoding.c

```diff
@@ -0,0 +1,75 @@
+/**
+ * @file run_length_encoding.c
```

Comment:
It's best not to include the name, otherwise, you'd have to include the full directory.
```suggestion
 * @file
```